calculator.py

```diff
@@ -37,48 +37,99 @@
     that explains how to perform calculations.
   * Commit and push your changes to your fork.
   * Submit a link to your Session03 fork repository!
-
-
 """
-
+import traceback
 
 def add(*args):
     """ Returns a STRING with the sum of the arguments """
-
-    # TODO: Fill sum with the correct value, based on the
+    # Fill sum with the correct value, based on the
     # args provided.
-    sum = "0"
+    args = [int(arg) for arg in args]
+    val = args[0] + args[1]
+    return str(val)
+
+# Add functions for handling more arithmetic operations.
+def subtract(*args):
+    args = [int(arg) for arg in args]
+    val = args[0] - args[1]
+    return str(val)
+
+def multiply(*args):
+    args = [int(arg) for arg in args]
+    val = args[0] * args[1]
+    return str(val)
 
-    return sum
+def divide(*args):
+    args = [float(arg) for arg in args]
+    val = args[0] / args[1]
+    return str(val)
```

Comment:
notice you have a lot of duplicated code here.  Perhaps this might work better:
```
import operator

def do_math(op, caster, *args):
    nums = map(caster, args)
    return str(op(args[0], args[1]))

def multiply(args):
    return do_math(operator.mul, int, args)

def divide(args):
    return do_math(operator.div, float, args)
```